browser/components/MarkdownEditor.js

```diff
@@ -142,6 +142,12 @@ class MarkdownEditor extends React.Component {
     this.renderPreview(this.props.value)
   }
 
+  handleKeyDown(e) {
+    if (this.state.status === 'CODE' && e.key == 'Escape') {
```

Comment:
Oh, I often use `Ctrl + E` in order to move cursor.... Considering it, I'm not sure it's the best.